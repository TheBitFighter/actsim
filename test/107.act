defproc test()
{
   int<6> i, j;

   int<12> k;

   chp {
     i := 1;
     j := 7;
     k := {j,i};
     log ("%b got ", k);
     k := {j,i+1};
     log ("%b got ", k);
     skip
   }
}
