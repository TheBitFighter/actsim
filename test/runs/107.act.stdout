[                  30] <>   got 000111000001
[                  40] <>   got 001110000010
